framework/src/base/ScalarCoupleable.C

```diff
@@ -84,6 +84,15 @@ ScalarCoupleable::coupledScalar(const std::string & var_name, unsigned int comp)
   return getScalarVar(var_name, comp)->number();
 }
 
+Order
+ScalarCoupleable::coupledScalarOrder(const std::string & var_name, unsigned int comp)
+{
+  if (!isCoupledScalar(var_name, comp))
+    return FIRST;
```

Comment:
That is what `test\src\auxkernels\CoupledScalarAux.C` is doing. After looking into `ScalarCoupleable::getDefaultValue()`, the order here should be `FEProblem::getMaxScalarOrder()`.